server/exe/Main.hs

```diff
@@ -53,3 +62,20 @@ serverOptions =
           <> Options.value 8081
           <> Options.metavar "INT"
       )
+    <*> Options.option
+      Options.str
+      ( Options.long "node-socket"
+          <> Options.short 's'
+          <> Options.help "Cardano Node socket path"
+          <> Options.showDefault
+          <> Options.value "./node/socket/node.socket"
+          <> Options.metavar "PATH"
+      )
+    <*> Options.option
+      networkIdReader
+      ( Options.long "network-id"
+          <> Options.help "Network id: either 'mainnet' or a NetworkMagic number"
+          <> Options.showDefault
+          <> Options.value C.Mainnet
```

Comment:
Using `Mainnet` as the default is conceptually simpler, but CTL in general doesn't actually support mainnet at the moment. I'm wondering if this should be changed to testnet?